src/faebryk/libs/exceptions.py

```diff
@@ -235,7 +255,7 @@ def nom_nom_nom(self, exc: T, original_exinfo):
             exceptions = [exc]
 
         for e in exceptions:
-            self.logger.log(self.to_level, str(e), exc_info=e)
+            self.logger.log(self.to_level, str(e), exc_info=e, extra={"markdown": True})
```

Comment:
Does this need the same tuple treatment as https://github.com/atopile/atopile/pull/761/files#diff-269e00d53db17645671d77c768728e2cb6240229264b966225af3886c81b2751R40?